src/Entity/Task.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * A "Task" is a scheduled job.
+ *
+ * @ORM\Table
+ * @ORM\Entity(repositoryClass="App\Repository\TaskRepository")
+ * @ORM\HasLifecycleCallbacks()
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class Task
+{
+    const ACTION_CLOSE_STALE = 1;
+    const ACTION_CLOSE_DRAFT = 2;
```

Comment:
it's ok, but these aren't used in this PR

Reply:
Correct. This is the teaser for upcoming PRs.